### ISSUE TITLE
Process Wrong Return

### ISSUE THREAD

**Issue by SquitchYT** [microsoft/node-pty#493]
## Environment details

- OS: Windows
- OS version: 10
- node-pty version: ^0.10.0

## Issue description
On Windows, the process return the name of the PTY

**Comment by Tyriar** [comment]
Yes on Windows it doesn't work, you would need to use another solution such as parsing the process tree [with this](https://www.npmjs.com/package/windows-process-tree) which is what vscode does.
